test/mjsonwp/mjsonwp-e2e-specs.js

```diff
@@ -467,15 +471,71 @@ describe('MJSONWP', async () => {
 
         it(`should pass with 200 HTTP status code if the command returns a value`, async () => {
           driver.performActions = (actions) => 'It works ' + actions.join('');
-          const {status, value} = await request.post(`${sessionUrl}/actions`, {
+          const {status, value, sessionId} = await request.post(`${sessionUrl}/actions`, {
             json: {
               actions: ['a', 'b', 'c'],
             }
           });
+          should.not.exist(sessionId);
           should.not.exist(status);
           value.should.equal('It works abc');
           delete driver.performActions;
         });
+
+        describe('jwproxy', function () {
+          let port = 56562;
+          let server, jwproxy, app;
+          beforeEach(function () {
+            let res = createProxyServer(sessionId, port);
+            server = res.server;
+            app = res.app;
+            jwproxy = new JWProxy({host: 'localhost', port});
+            jwproxy.sessionId = sessionId;
+            driver.performActions = async (actions) => await jwproxy.command('/perform-actions', 'POST', actions);
+          });
+
+          afterEach(function () {
+            server.close();
+            delete driver.performActions;
+          });
+
+          it('should work if a proxied request returns a response with status 200', async () => {
```

Comment:
async function

Reply:
same below

Reply:
They have `await` in them. If you're talking about `addHandler` the callback actually isn't async. It's an Express handler.

Reply:
he means that in mocha tests we should try to write `async function () {` instead of `async () => {`, since mocha needs binding to work properly. it usually doesn't matter and i'm on the fence about making this a requirement, but it's not a bad idea.

Reply:
Aha, yeah it probably should be a requirement. That's bitten me in the past a few times.

Reply:
I think there's an ESLint rule for that.

Reply:
https://github.com/appium/eslint-config-appium/pull/12